src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -181,6 +181,9 @@
     @Argument(doc = "INFO field annotations that should be deleted when swapping reference with variant alleles.", optional = true)
     public Collection<String> TAGS_TO_DROP = new ArrayList<>(LiftoverUtils.DEFAULT_TAGS_TO_DROP);
 
+    @Argument(doc = "Output VCF file will be written on the fly but it won't be sorted and indexed.", optional = true)
```

Comment:
```suggestion
    @Argument(doc = "Output VCF file will be written on the fly but it won't be sorted nor indexed.", optional = true)
```

Reply:
“or” actually :smile:

Reply:
I'm not native english speaker, nor do I care that much :-D